ChangeLog.md

```diff
@@ -1,11 +1,15 @@
 # Adobe Branch SDK Extension change log
 
+- v2.1.0
+  * March 18, 2023
+  * Update Branch Android SDK to 5.10.1
```

Comment:
Update to current for both ios and android

Reply:
@gdeluna-branch Just updated to the android sdk to 5.11.0

Reply:
And the current date